File: CPP/two_sum.cpp
class Solution {
    public:
        vector<int> twoSum(vector<int> &numbers, int target) {
            // IMPORTANT: Please reset any member data you declared, as
            // the same Solution instance will be reused for each test case.
            vector<int> result;
            for (int i = 0; i < numbers.size(); i++) {
                for (int j = i + 1; numbers[i] <= target && j < numbers.size(); j++) {
                    if (numbers[j] <= target && numbers[i] + numbers[j] == target) {
                        result.push_back(i + 1);
                        result.push_back(j + 1);
                        break;
                    }
                }
            }
            return result;
        }
};

File: JAVA/TwoSum.java
public class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int index1 = 0, index2 = 0;
        int size = numbers.length;
        for (int i = 0; i < size && index2 == 0; i++) {
            for (int j = i + 1; j < size; j++) {
                if (numbers[i] + numbers[j] == target) {
                    index1 = i + 1;
                    index2 = j + 1;
                    break;
                }
            }
        }
        int[] result = {index1, index2};
        return result;
    }
}